WordPress/Sniffs/WP/DeprecatedFunctionsSniff.php

```diff
@@ -334,7 +349,7 @@ class WordPress_Sniffs_WP_DeprecatedFunctionsSniff extends WordPress_AbstractFun
 			'version' => '2.8.0',
 		),
 		'get_the_author_description' => array(
-			'alt'     => 'get_the_author_meta( \'description\' )',
+			'alt'     => 'get_the_author_meta(\'description\')',
```

Comment:
Yes, the spacing of the various alternatives is currently inconsistent and inconsistent with the WP coding standard.
Consistency with WP core has been given preference for easier comparisons.
Also see: https://core.trac.wordpress.org/ticket/41121#comment:10